cfme/configure/tasks.py

```diff
@@ -128,7 +135,8 @@ class tabs(View):  # noqa
 
         @View.nested
         class mytasks(Tab):  # noqa
-            TAB_NAME = "My VM and Container Analysis Tasks"
+            TAB_NAME = VersionPick({Version.lowest(): 'My VM and Container Analysis Tasks',
```

Comment:
On second consideration, until this module is refactored appropriately to get rid of the module methods, this is actually allow-able and covers tests/framework where `mytasks` is used instead of `myothertasks`, where in 59z they both should direct to 'My Tasks' named tab.